tests/framework/common.go

```diff
@@ -346,14 +355,21 @@ func (td *OsmTestData) LoadImagesToKind(imageNames []string) error {
 	if err != nil {
 		return errors.Wrap(err, "failed to get image data")
 	}
+
+	imageReader, _ := ioutil.ReadAll(imageData)
```

Comment:
Let's handle this error just to be safe, though I don't expect we'll ever hit it.